src/sentry/utils/sdk_crashes/sdk_crash_detector.py

```diff
@@ -63,6 +63,9 @@ def should_detect_sdk_crash(
         if is_fatal and self.config.report_fatal_errors:
             return True
 
+        if mechanism_type in ["AppExitInfo", "ANR"]:
```

Comment:
@philipphofmann do you also want to track app hangs actually? might be noisy though

Reply:
On iOS, this would be crazy noisy. Maybe we can enable this once we have the new algorithm shipped to all users.

Reply:
> I think we should add this to the config and only specify this for mechanism_type for Android for now, cause it's Android related.

extracted into the config in https://github.com/getsentry/sentry/pull/82030/commits/bed3383fb7b2d5009b44d642fe2c1ba0a5859529